src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -324,42 +311,17 @@ The `@aws-amplify/ui-vue` package is a set of components that make it easy to in
 </Block>
 </BlockSwitcher>
 
-#### Troubleshooting
-[question]: # (RENE, Any content for the troubleshooting section?)
-
-[comment]: # (EXPANDER SECTION to provide tips on how to avoid common errors such as: not changing the default auth rules, CLI set up correctly, and troubleshooting an AWS account with the CLI. Each error should have a header with descriptive title to be included on a primary troubleshooting guide as well.)
-  
-[comment]: # (EXPANDER end)
-
-### Recap
-Well done! You now have your Amplify Libraries set up and configured. 
-
-[comment]: # (ADD TO RECAP tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
+**Recap:** Well done! You now have your Amplify Libraries set up and configured. 
 
 ## Create, update, and delete application data
-In this job, you will run mutations to create, update, and delete application data. We will also show you how to cancel these requests. 
 
-> **Before you begin, you will need:**
-> - Your application connected to the API
+In this job, you will run mutations to create, update, and delete application data. We will also show you how to cancel these requests. Before you begin, you will need your application connected to the API.
 
 ### Run mutations to create, update, and delete application data
   
-[comment]: # (Short intro to review mutations with the Amplify GraphQL client.)
-  
-In GraphQL, mutations write data to the API and are used to create, update, or delete data. 
-
-#### Learn more about GraphQL mutations
-
-[comment]: # (EXPANDER SECTION that covers GraphQL basics, mutations, and why this is helpful—can be a short summary with recommended links.)
-  
-[comment]: # (EXPANDER end)
-[comment]: # (RENE, what text are we planning to use within the "Learn more about GraphQL mutations" expander? We're missing a high-level overview of the basics and the benefits of mutations.)
-
-Here are some examples of how you can create, update, and delete items using the Amplify GraphQL client.
+In GraphQL, mutations write data to the API and are used to create, update, or delete data. This is different than queries that allow you to read the data but not change it. What follows are some examples of how you can create, update, and delete items using the Amplify GraphQL client.
```

Comment:
Note here that I thought an expander seemed like too much to remind them of the diff b/w mutations and queries. I added the brief review directly into this intro. We can change it back to something more robust if needed.

Reply:
Acknowledged. This works.